packages/big-design/src/components/Dropdown/types.ts

```diff
@@ -1,14 +1,16 @@
 import { Placement } from '@popperjs/core';
 import { LiHTMLAttributes } from 'react';
 
+import { ButtonProps } from '../Button';
+
 export interface DropdownProps extends Omit<React.HTMLAttributes<HTMLUListElement>, 'children'> {
   autoWidth?: boolean;
   disabled?: boolean;
   items: Array<DropdownItem | DropdownLinkItem> | DropdownItemGroup[];
   maxHeight?: number;
   placement?: Placement;
   positionFixed?: boolean;
-  toggle: React.ReactElement;
+  toggle: React.ReactElement<ButtonProps>;
```

Comment:
Would this be `ButtonProps` or actually `unknown`? We don't actually enforce this as a `Button` element, so it could be anything.